.gitignore

```diff
@@ -1,14 +1,20 @@
 # Generated by Cargo
 # will have compiled files and executables
-debug/
-target/
+**/debug/
+**/target/
 
 # Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
 # More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
-Cargo.lock
+**/Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-# MSVC Windows builds of rustc generate these, which store debugging information
-*.pdb
+# VS code folder
+.vscode/
+
+# github folder
+.github/
```

Comment:
Why are we ignoring the `.github` folder? This will contain all the workflows. Don't you think we should be keeping them in the backed-up repository?